cmd/agent/app/status.go

```diff
@@ -121,6 +133,40 @@ func requestStatus() error {
 	return nil
 }
 
+// getAPMStatus returns a set of key/value pairs summarizing the status of the trace-agent.
+// If the status can not be obtained for any reason, the returned map will contain an "error"
+// key with an explanation.
+func getAPMStatus() map[string]interface{} {
+	port := 8126
```

Comment:
Once I clean up the below (I already have a task in our backlog), this will be removed anyway. I think there's no reason to add this constant in the config package and yet another place where to clean up after that task is done.